doc/lag/Increasing LACP PDU timeout during warm-reboot.md

```diff
@@ -0,0 +1,119 @@
+# Introduction
+## Overview
+
+During warm-reboot, the control plane can be down for a maximum of 90 seconds.
+This is beacuse LACP PDUs are sent every 30 seconds, and the protocol allows for
+up to 3 LACP PDUs to be missed before the LAG is considered down and data
+traffic is disrupted.
+
+It would be beneficial if it's possible to temporarily increase the timeout for
+LACP PDUs on a LAG on both sides. Specifically, prior to starting warm-reboot,
+the timeout could be increased by some amount (beyond the limits of the
+protocol), and after warm-reboot, the timeout would be restored to the normal
+value.
+
+## Requirements
+
+- Switch running a supported SONiC with patches in libteam for this feature on
```

Comment:
Will this feature (patch) stay as patch in libteam specifically for SONiC or will be pushed to libteam community as well?

Reply:
Because this is effectively breaking the LACP protocol, I'm not planning on submitting this patch upstream.